### ISSUE TITLE
GDB doesn't support CUSTOM0/cfu instruction on disassemble command.

### ISSUE THREAD

**Issue by mgaggero** [google/CFU-Playground#733]
Hi,
    the `disassemble [/r|/m|/s]` command on gdb running against cfu code does not show the CUSTOM0/CFU instruction mnemonic but its machine code representation.

The toolchain is the August 2020 toolchain from freedom-tools.
GDB version is GNU gdb (SiFive GDB 9.1.0-2020.08.2) 9.1.
GCC version is 10.1.0 (SiFive GCC 10.1.0-2020.08.2).

Maybe related to #470.

![gdb_custom0](https://user-images.githubusercontent.com/6576853/200186308-22af375c-85e0-4244-b94a-128e45243a0b.png)

**Comment by tcal-x** [comment]
@mgaggero yes you are correct on all counts.   We do have a special disassembly script that patches that, but it doesn't help with GDB.    I haven't had time to follow up on #470 and am not sure when I can, but thanks for raising awareness that it would be useful!

**Comment by tcal-x** [comment]
@PiotrZierhoffer , @mateusz-holenko , @kgugala -- do you have any experience with this from your other work with RISC-V custom extensions?    See the link in #470.
